ts/store/reducers/persistedPreferences.ts

```diff
@@ -20,14 +21,18 @@ export type PersistedPreferencesState = Readonly<{
   wasServiceAlertDisplayedOnce?: boolean;
   isPagoPATestEnabled: boolean;
   isExperimentalFeaturesEnabled: boolean;
+  // TODO: create transformer for Option objects and use Option instead of pot
+  //       https://www.pivotaltracker.com/story/show/170998374
+  isCustomEmailChannelEnabled: pot.Pot<boolean, undefined>;
```

Comment:
```suggestion
  isCustomEmailChannelEnabled?: pot.Pot<boolean, undefined>;
```
it should be optional: for a generic user this part of the store can already exists (_PersistedPreferencesState_). In that case the **initialPreferencesState** will never returned by the reducer. So **isCustomEmailChannelEnabled** will be undefined

Make it optional could be a solution.
Another solution it's make a migration that set the initial value of the field **_isCustomEmailChannelEnabled_**

Reply:
Let me know if this logic satisfy your case: at startup, we fork the `checkEmailNotificationPreferencesSaga`: it checks if the state `isCustomEmailChannelEnabled` has a valid value and, otherwise, it evaluates the correct value for `isCustomEmailChannelEnabled`.